manager.go

```diff
@@ -38,6 +38,14 @@ const (
 	LabelSourceType = "cloudfoundry.org/source_type"
 )
 
+type WatcherChannelClosedError struct {
```

Comment:
This needs  an  annotation in comments with a bit of description, at least will get a godoc  entry